### DISCUSSION TITLE
Team join request

### DISCUSSION THREAD

**Discussion by Lambada10** [ROM-builders/temporary#11491] [Q&A]
Build: https://cirrus-ci.com/task/5034382296612864

Trees:
https://github.com/ProjectBlaze-Devices/device_xiaomi_vayu
https://github.com/ProjectBlaze-Devices/vendor_xiaomi_vayu
https://github.com/Jebaitedneko/android_kernel_xiaomi_vayu

Telegram: @Lambada10

**Comment by Apon77** [comment]
please follow #learn from telegram group

**Reply by Lambada10** [reply]
I know how to do these

**Reply by Apon77** [reply]
> I know how to do these

your profile does not say this

**Reply by Lambada10** [reply]
Show me what's wrong
